packages/components/addon/components/hds/form/field/index.js

```diff
@@ -3,7 +3,18 @@ import Component from '@glimmer/component';
 import { guid } from '../utils/guid';
 import { describedBy } from '../utils/describedby';
 
-export default class HdsFormRadioFieldComponent extends Component {
+export default class HdsFormFieldIndexComponent extends Component {
+  /**
+   * Sets the layout of the group
+   *
+   * @param layout
+   * @type {enum}
+   * @default 'vertical'
+   */
+  get fieldLayout() {
```

Comment:
since it's implicitly a `field` maybe I would call it just `@layout`?

Reply:
Update from `fieldLayout` to `layout`